concent_api/conductor/migrations/0008_auto_20180717_0908.py

```diff
@@ -23,11 +23,13 @@ class Migration(migrations.Migration):
     dependencies = [
         ('conductor', '0007_auto_20180709_1526'),
     ]
+
     operations = [
         migrations.AddField(
             model_name='verificationrequest',
             name='created_at',
-            field=models.DateTimeField(default=django.utils.timezone.now),
+            field=models.DateTimeField(auto_now_add=True, default=django.utils.timezone.now),
```

Comment:
> The options `auto_now_add`, `auto_now`, and `default` are **mutually exclusive**. Any combination of these options will result in an error.

Choose one ;) (https://docs.djangoproject.com/en/2.0/ref/models/fields/#django.db.models.DateField)

Reply:
See:
```
I have discovered something strange. If you want to add default to field definition you can't do this when you have auto_now or auto_now_add option.

(fields.E160) The options auto_now, auto_now_add, and default are mutually exclusive. Only one of these options may be present.

but it is impossible to add field to existing model without adding default value and it is necessary to add value during makemigrations command - and there, as a default is timezone.now
``` 
I have wrote this above because I wanted to do this without default and Django didn't let me to do this and during `makemigrations` command Django made suggestion to add `default` with `timezone.now`

Reply:
I have tested locally with old database and records in it and then I applied migrations and everything worked.

Reply:
Just a hint for the future: we never want fake defaults in migrations. If you think you need one, this needs to be discussed first. You may end up making irreversible damage to the data in production otherwise.